web/template/layout/case-details.gohtml

```diff
@@ -0,0 +1,3 @@
+{{ define "case-details" }}
+    <p class="govuk-body"><strong>{{ if eq .Case.SubType "pfa" }}PFA{{ else if eq .Case.SubType "lpa" }}LPA{{ end }} {{ .Case.UID }}</strong></p>
```

Comment:
I'm not sure when this slipped in but this should surely be saying PFA or HW?

Reply:
As far as i can tell it has always been pfa or lpa... i can double check when everyone is back